### ISSUE TITLE
v2.0 Build-issue: Missing shader

### ISSUE THREAD

**Issue by agonio** [Iswenzz/CoD4-Battleroyale#4]
### Error description
Linker-error running `scripts/build.sh`

```
********************************************************************************
UNRECOVERABLE ERROR:
  Can't find shader: shader_bin/vs_3_0_00000000
********************************************************************************
```

### Repro steps
- (preinstalled the Mod-Tools)
- Download the sources from release 2.0 into the \<cod4>/Mods dir (dir: sr_battleroyale)
- run build scripts as described in the readme. 
  - `scripts/build_iwds.sh` ✔ 
  - `scripts/build.sh` ❌

### Details

I noticed the following line in the log: `cp: cannot stat 'shader_bin': No such file or directory`. The build script tries to copy from a local `sr/assets/shader_bin` folder, which is not present in the release sources without access to the private submodule.
I assume that there's a reason, why the submodule is private and people won't just get access to it.

### Suggestion
I think the easiest solution would be (especially for non-devs) to add an archive with prebuilt iwds and mod.ff to the release attachements.
If this is somehow not desirable, would it be possible to provide the necessary assets on another way? (Pretty sure that the shader won't be the only missing dependency in the end)

**Comment by agonio** [comment]
\* Maybe one addition why I'm trying to setup your mod: we're making a LAN-Party by the end of the year and I would like to breathe new life into our cod4 session.

**Comment by Iswenzz** [comment]
The private module is for SR private servers, it is not required for the public release, i should probably silent the copy warnings. i don't think the shaders are missing, the shader error is probably above, i suspect you need admin privilege to run the shader tool.

**Comment by agonio** [comment]
Thanks for your fast response, I understand.
I actually had some trouble with the shader-tool in the beginning. It was missing the `ucrtbased.dll` to run. I found out, that the debug build of that dll is only shipped with visual studio.
I checked again, and I must've messed up the shader_tool/dll indeed:
![grafik](https://github.com/Iswenzz/CoD4-Battleroyale/assets/9164826/39023893-d6c0-4c3b-8682-9ce1498471a1)

Interesting though, that it does not report an error, although it's a non-zero exit code.
Thanks for pointing me in that direction.
